features/upgrade/etcd/upgrade.feature

```diff
@@ -13,26 +12,51 @@ Feature: basic verification for upgrade testing
   @arm64 @amd64
   Scenario: etcd-operator and cluster works well after upgrade - prepare
     Given I switch to cluster admin pseudo user
-    Given I obtain test data file "admin/subscription.yaml"
-    When I run the :create client command with:
-      | f | subscription.yaml |
-    Then the step should succeed
-    When I use the "openshift-operators" project
-    Then status becomes :running of exactly 1 pods labeled:
-      | name=etcd-operator-alm-owned |
-    When I use the "default" project
-    Given I obtain test data file "admin/etcd-cluster.yaml"
-    When I run the :create client command with:
-      | f | etcd-cluster.yaml |
-    Then the step should succeed
-    #bugzilla - id=1979550
-    #Then status becomes :running of exactly 3 pods labeled:
-    #  | etcd_cluster=example |
+    Given I store the masters in the :masters clipboard
+    When I use the "openshift-etcd" project
+    When I run the :get admin command with:
```

Comment:
@geliu2016 @sandeepknd you can replace the :get command (line 17-22) with the following
```
    And I store in the :pods clipboard the pods labeled:
      | app=etcd |
    Then the expression should be true> cb.pods.select {|p| p.ready_state_reachable?}.count == cb.masters.count
```